jest.config.js

```diff
@@ -17,8 +15,6 @@ module.exports = {
             statements: 80,
         },
     },
-    // Direct Jest to read the jest config file from integration tests
-    projects: ['<rootDir>', integrationTestPath],
```

Comment:
Would this still read the jest config in integ test (with diff timeout etc) ?

Reply:
Yes